modules/ting/ting.entities.inc

```diff
@@ -298,7 +298,19 @@ class TingEntity extends DingEntity {
    *   The language, FALSE if it could not be found.
    */
   public function getLanguage() {
-    return empty($this->reply) ? FALSE : $this->reply->getLanguage();
```

Comment:
Are you sure this does not belong in https://github.com/easySuite/ding2/blob/development-4.4.x/modules/opensearch/src/OpenSearchTingObject.php#L202?

Reply:
Please relate to my comment.

Reply:
I mean; are you sure that the code you have added should be added in `ting/ting.entities.inc` and not in `opensearch/src/OpenSearchTingObject.php` instead?

Reply:
No. Because in a final we are processing TingEntity object which is going through TingEntity class declared in ting/ting.entities.inc.